lib/public/views/Runs/ActiveColumns/getQcSummaryDisplay.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+import { h, iconWarning, iconBolt } from '/js/src/index.js';
+import { formatFloat } from '../../../utilities/formatting/formatFloat.js';
+import { getValueFromCssVar, isContrastBlack } from '../../../components/common/badge.js';
+import { QC_SUMMARY_COLORS } from '../../../components/qcFlags/qcSummaryColors.js';
+import { tooltip } from '../../../components/common/popover/tooltip.js';
+
+const FULL_COVERAGE = 1;
+
+/**
+ * Render display common for QC and GAQ summaries
+ *
+ * @param {RunDetectorQcSummary|RunGaqSummary} summary summary
+ * @return {Component} display
+ */
+export const getQcSummaryDisplay = (summary) => {
+    const {
+        badEffectiveRunCoverage,
+        explicitlyNotBadEffectiveRunCoverage,
+        missingVerificationsCount,
+        mcReproducible,
+    } = summary;
+
+    const missingVerificationMessage = `Missing ${missingVerificationsCount} verification${missingVerificationsCount > 1 ? 's' : ''}`;
```

Comment:
As this `const` is only used once in the ternary condition and only if there are missing verifications count, I would say the message can be declared directed in the `h()` method

Reply:
fixed